contrib/func_tests.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/zsh
+# The following script runs three known test cases for qtop, one for each of PBS, OAR, SGE
+# No output after "Testing <scheduler>" means test passed. 
+# The test actually runs qtop over known scheduler output files, and then diffs them against the expected output. 
+# While diffing, one qtop output line is omitted 
+# (the one containing word "WORKDIR"), as it contains an everchanging timestamp.
+# WARNING: this will only run in zsh, not bash
+
+export QTOPDIR="/home/sfranky/PycharmProjects/qtop"
+alias -g NF='*(.om[1])' # newest file in a directory
+
+alias sger='$QTOPDIR/qtop.py -s $QTOPDIR/contrib -FadvvO -b sge'
+alias dsger="diff <(grep -v 'WORKDIR' NF) <(grep -v 'WORKDIR' $QTOPDIR/contrib/sger_dvv_out.ref)"
+alias oar1='$QTOPDIR/qtop.py -s $QTOPDIR/contrib -FardvvvOA -b oar'  # anonymised test
+alias doar1="diff <(grep -v 'WORKDIR' NF) <(grep -v 'WORKDIR' $QTOPDIR/contrib/oar1_anon_dvv_out.ref)"
+alias 4tab='$QTOPDIR/qtop.py -s $QTOPDIR/contrib -raFO -b pbs'
+alias d4tab="diff <(grep -v 'WORKDIR' NF) <(grep -v 'WORKDIR' $QTOPDIR/contrib/pbs_dvv_out.ref)"
+
+
+cd /tmp/  # qtop output is saved there
+echo "Testing sge..."
+sger
+sleep 1
```

Comment:
weird; if you try multiple runs with no delay, can you trace if/when things become normal? (run it x10 times without a sleep)